tap-snapshots/test/lib/commands/shrinkwrap.js.test.cjs

```diff
@@ -46,6 +47,9 @@ exports[`test/lib/commands/shrinkwrap.js TAP with hidden lockfile ancient upgrad
   },
   "logs": [
     "created a lockfile as npm-shrinkwrap.json with version 3"
+  ],
+  "warn": [
+    "Converting lock file (npm-shrinkwrap.json) from v1 -> v3"
```

Comment:
Were we just not asserting/looking for this before?

Reply:
I don't know the exact reason (most likely the way logs were being mocked) but warnings were never making it into the logs. I'm assuming when I wrote those tests I was just asserting the current behavior. But now I can confirm that those warnings are real and being shown to users in all those cases

Reply:
Yep this is a good signal here. Resolving this thread.